client/src/app/pages/assessment-management/questionnaire/questionnaire-page.tsx

```diff
@@ -40,34 +41,28 @@ const Questionnaire: React.FC = () => {
     setActiveSectionIndex(tabKey as "all" | number);
   };
 
-  const [assessmentData, setAssessmentData] = useState<YamlAssessment | null>(
-    null
-  );
-
-  // ------------------------!!
-  // TODO: replace this with the real data from the API
-  const fetchError = undefined;
-  const isLoading = false;
+  const [assessmentData, setAssessmentData] = useState<Assessment | null>(null);
 
-  useEffect(() => {
-    fetch("/questionnaire-data.yaml") // adjust this path
-      .then((response) => response.text())
-      .then((data) => {
-        const parsedData = yaml.load(data) as YamlAssessment;
-        setAssessmentData(parsedData);
-      });
-  }, []);
-  // ------------------------!!
+  // useEffect(() => {
+  //   fetch("/questionnaire-data.yaml") // adjust this path
+  //     .then((response) => response.text())
+  //     .then((data) => {
+  //       const parsedData = yaml.load(data) as Assessment;
+  //       setAssessmentData(parsedData);
+  //     });
+  // }, []);
+  // // ------------------------!!
```

Comment:
I think it's safe to just drop this comment block and let it source from MSW data (until hub can do it from main)